addons/web/static/src/core/position_hook.js

```diff
@@ -308,6 +308,6 @@ export function usePosition(target, options) {
                 window.removeEventListener("resize", listener);
             };
         },
-        () => [getTarget(), popperRef.el]
+        () => [getTarget(), popperRef.el, popperRef.el?.innerHTML]
```

Comment:
From the positioning hook perspective, it does not have to recompute when the innerHTML of the popper changes.

So instead of doing this, could you please do:
```suggestion
```

Then in the first line of the callback passed to this useEffect:
```js
const targetEl = getTarget();
const popperEl = popperRef.el;
```